docs/public/specifications/tests/Address-TP/address02.md

```diff
@@ -1,46 +1,102 @@
-## ADDRESS02: Reverse DNS entry exists for name server IP address
+# ADDRESS02: Reverse DNS entry exists for name server IP address
```

Comment:
The head line is quite long. What do you think of the following, shorter alternatives?
* Reverse DNS entry for name server IP
* PTR exists for name server IP